src/gpodder/model.py

```diff
@@ -94,6 +96,7 @@ class EpisodeModelFields(minidb.Model):
     chapters = minidb.JSON
     subtitle = str
     description_html = str
+    episode_art_url = str
```

Comment:
Shouldn't there be a db migration? Or maybe it's not needed in gpodder-core (I didn't check)?

Reply:
I kind of expected that but the minidb magic @thp created does it on the fly as far as I can tell.
During startup it runs an alter table for missing columns and I think it does not delete upon revert (I did not test that because I haven't needed to revert back to master so far).

Reply:
After reverting to master it seems that it just ignores the extra column, doesn't remove it or delete its' contents.